testdata/golang/light/knapsack.go

```diff
@@ -1,34 +1,37 @@
 package light
 
-import "math"
+import (
+	"errors"
+	"math"
+)
 
-type Item struct {
-	weight int
-	value  int
-}
```

Comment:
I think the Go example was fine. Did not cause any mistakes. LLMs should be able to deal with structs as arguments...